src/raw/error.rs

```diff
@@ -44,7 +42,7 @@ pub enum ErrorKind {
     MalformedValue { message: String },
 
     /// Improper UTF-8 bytes were found when proper UTF-8 was expected.
-    Utf8EncodingError(Utf8Error),
```

Comment:
The error value provided by `simdutf8::basic::from_utf8` is zero-sized (because it omits tracking error location in favor of speed) so I didn't see much reason to capture it here; this also means we're not exposing a 0.x crate type in our public API.